graylog2-rest-client/src/main/java/org/graylog2/restclient/models/Stream.java

```diff
@@ -39,8 +39,9 @@
 import java.util.List;
 import java.util.Map;
 
-public class Stream {
+import static com.google.common.base.Strings.emptyToNull;
```

Comment:
I couldn't find any usages of `emptyToNull` on this class, can we get rid of this import?